Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM timbru31/node-alpine-git:16 as builder
+FROM timbru31/node-alpine-git as builder
 
 WORKDIR /tmp/nexus-web
 COPY . /tmp/nexus-web
 ENV GENERATE_SOURCEMAP=false
```

Comment:
we can remove `GENERATE_SOURCEMAP`, vite don't use this env

Reply:
Really good point. I'll remove this.